virt-who-config.5

```diff
@@ -145,6 +145,9 @@ Path to Kubernetes configuration file which contains authentication and connecti
 .TP
 \fB#kubeversion\fR
 API version used to override kubevirt api version fetched from the cluster. Used by kubevirt option
+.TP
+\fB#insecure\fR
+Allows not to validate tls certificates during connection to kubevirt
```

Comment:
Eliminate validation of tls certificates ...

Reply:
Done